pyproject.toml

```diff
@@ -36,6 +36,7 @@ pytz = "==2023.3.post1"
 rtreelib = "==0.2.0"
 werkzeug = "^3.0.1"
 wtforms = "~=3.1"
+poetry-plugin-dotenv = "^0.5.3"
```

Comment:
Flagging this as something that I'm going to remove once this is merged because it's addressed in the project setup updates and shouldn't be added directly to the project dependencies.

Reply:
Forcing GitHub to resolve the comment now!